.travis.yml

```diff
@@ -11,5 +11,9 @@ before_install:
   - docker ps -a
 before_script:
   - npm install -g gulp
+  - gulp selenium-install
   - ./node_modules/db-migrate/bin/db-migrate up
-script: gulp test && gulp test-integration
+script: 
```

Comment:
the second commit makes it more : https://github.com/lplotni/pace/pull/253/commits/d5db0058b29afc950bb6d2c694b548b3d4060f9d